sources/ch/luzern.json

```diff
@@ -0,0 +1,15 @@
+{
+    "coverage": {
+        "country": "ch",
+        "city": "Luzern"
+    },
+    "data": "https://svc-test.geo.lu.ch/agssec/rest/services/basis/basis_gbplan/MapServer/49",
+    "type": "ESRI",
+    "conform": {
+        "type": "geojson",
+        "number": "HAUSNUMMER",
+        "street": "LOK_NAME",
+        "city": "ORTSCHAFT",
```

Comment:
For some of the Swiss data sets the canton abbreviation is added onto the end of the city name ("LU" in this case). This addition will remove it:

```json
        "city": {
            "function": "regexp",
            "field": "ORTSCHAFT",
            "pattern": "^(.+?)(?:\\s+LU)?\\s*$"
        },
```